### PR TITLE
React client added, README.md updated, Node.js fix

### PR DESCRIPTION
**Pull request by nidhinpd-YML**
- React client added
   - Supports APIs `chat/` and `stream/`
- General README added
- Node.js fix, default array in request history of stream API

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,245 @@
...
+<code>POST</code> <code><b>chat/</b></code>
+<h4>Parameters</h4>
+<table class="responsive fixed orange">
+<tr><td><h5>Name</h5></td><td><h5>Type</h5></td><td><h5>Data type</h5></td><td><h5>Description</h5></td></tr>
+<tr><td>chat</td><td>required</td><td>string</td><td>Latest chat message from user</td></tr>
+<tr><td>history</td><td>optional</td><td>array</td><td>Current chat history between user and Gemini model</td></tr>
+</table>
+<h4>Response</h4>
+<table class="responsive fixed orange">
+<tr><td><h5>HTTTP code </h5></td><td><h5>Content-Type</h5></td><td><h5>Response</h5></td></tr>
```
Comment (pcoet) [inline README.md:61]: typo: "HTTTP" -> "HTTP"; looks like it happens later, too

File: README.md [hunk 2]
```diff
@@ -1,5 +1,245 @@
...
+
+You can choose to install the required packages for
+- Frontend(React) plus a backend server of your choice(Flask or Node.js).
+- Frontend(React) and both backend servers(Flask, and Node.js).
+
+You can specify your choice using the `BACKEND` variable while running the script.
+
+Install React along with Node.js backend.
+```
+BACKEND=nodejs ./setup.sh
```
Comment (pcoet) [inline README.md:109]: Does this need the `source` command too? Just noticed the other two install commands have it.

File: client-react/public/logo192.png
Comment (pcoet) [inline client-react/public/logo192.png:1]: Are we actually using this image? It'd be better to leave it out, if we don't need. Same with other logos.

**Review by pcoet — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (nidhinpd-YML) [reply README.md:109]: No. Source is only needed for Python venv setup. When BACKEND=nodejs, it is not needed.

**Pull request merged**